src/main/java/com/elixr/poc/rest/validator/UserValidator.java

```diff
@@ -0,0 +1,27 @@
+package com.elixr.poc.rest.validator;
+
+import com.elixr.poc.constant.ErrorConstants;
+import com.elixr.poc.rest.request.UserRequest;
+import lombok.Builder;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Builder
+public class UserValidator {
+    public List<String> validate(UserRequest newUserRequest) {
+        List<String> errorList = new ArrayList<>();
+        if (newUserRequest.getFirstName() == null || newUserRequest.getFirstName().isEmpty()) {
```

Comment:
You can use StringUtils.isBlank(newUserRequest.getFirstName())  it will check for both null and empty.Do the same for all validation

Reply:
Updated and using StringUtils.isBlank() for validation.